indico/modules/vc/templates/vc_room_list.html

```diff
@@ -17,7 +17,7 @@
         </td>
         <td>
             <div class="title">
-                {% trans %}VC room{% endtrans %}
+                {% trans %}Meeting room{% endtrans %}
```

Comment:
This sounds somewhat confusing. "VC name" maybe?

Reply:
Not sure if "videoconference name" is applicable here since more details are shown. What about just "Videoconferences"?

Reply:
sounds fine as well